cookbook/attributes/agent.rb

```diff
@@ -0,0 +1,6 @@
+default[:baragon][:group_name] = 'default'
+default[:baragon][:proxy_conf_dir] = '/tmp'
+default[:baragon][:upstream_conf_dir] = node[:baragon][:proxy_conf_dir]
```

Comment:
I recall first encountering this with the Zookeeper cookbook, and it having the ZK version, and the URL to download ZK (which included the version), being defined together.

End result was that, with the version being overridden in a wrapper, the URL would still end up including the original attribute, so the wrapper had to define _both_ attributes.